app/templates/logo.hbs

```diff
@@ -114,6 +114,56 @@
             <figcaption class="text-small"><a href="images/logos/TravisCI-Mascot-red.eps" download="TravisCI-Mascot-red.eps" title="">.eps</a> <a href="images/logos/TravisCI-Mascot-red.png" download="TravisCI-Mascot-red.png" title="">.png</a></figcaption>
           </figure>
         </li>
+        <li>
+          <figure>
+            <img src="../images/logos/TravisCI-Mascot-pride.svg" alt="Mascot">
+            <figcaption class="text-small"><a href="images/logos/TravisCI-Mascot-pride.eps" download="TravisCI-Mascot-pride.eps" title="">.eps</a> <a href="images/logos/TravisCI-Mascot-pride.png" download="TravisCI-Mascot-pride.png" title="">.png</a></figcaption>
+          </figure>
+        </li>
+        <li>
+          <figure>
+            <img src="../images/logos/TravisCI-Mascot-pride-4.svg" alt="Mascot">
+            <figcaption class="text-small"><a href="images/logos/TravisCI-Mascot-pride-4.eps" download="TravisCI-Mascot-pride-4.eps" title="">.eps</a> <a href="images/logos/TravisCI-Mascot-pride-4.png" download="TravisCI-Mascot-pride-4.png" title="">.png</a></figcaption>
+          </figure>
+        </li>
+      </ul>
+      <ul class="list--tessa">
```

Comment:
@lislis and I looked this over at the conference and since the `.list--tessa` Sass is exactly the same as `.list--mascot`, it seems like these `<li>` can just be in the same `<ul>`. Does that make sense? Then they can all be `.list--mascot` and no need to have the duplicated Sass.